apps/nextjs-example/src/components/transactionFlows/MultiAgent.tsx

```diff
@@ -78,8 +83,16 @@ export function MultiAgent() {
     if (!transactionToSubmit) {
       throw new Error("No Transaction to sign");
     }
+    if (!secondarySignerAccount) {
+      throw new Error("No secondarySignerAccount");
+    }
     try {
-      const authenticator = await signTransaction(transactionToSubmit);
+      const authenticator = aptosClient(
```

Comment:
what this change for? the idea of using this demo app is to use the wallet adapter to sign a transaction and not the sdk